compose/frontend/default.conf

```diff
@@ -76,7 +76,7 @@ server {
 
     # Caching data from the API
     location /api/ {
-        proxy_ignore_headers Vary;
+        proxy_ignore_headers Vary; #This will need to be removed when not all API's are public anymore.
```

Comment:
It'll have to remain for non-user specific endpoints.  User-specific endpoints will need to not be cached or to respect the vary header.